### PR TITLE
Adding single Property Group get

### PR DESCRIPTION
**Pull request by Spookytheboy**
Adding get function for singular Contact Property Group.

### CODE REVIEW

**Comment by Spookytheboy** [comment]
No clue what this failure means.

File: src/Resources/ContactProperties.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ class ContactProperties extends Resource
      */
     public function all()
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/properties';
+        $endpoint = "https://api.hubapi.com/contacts/v2/properties";
```
Comment (ksvirkou) [inline src/Resources/ContactProperties.php]: Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)

File: src/Resources/ContactProperties.php [hunk 2]
```diff
@@ -51,7 +51,7 @@ public function get($name)
      */
     public function create(array $property)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/properties';
+        $endpoint = "https://api.hubapi.com/contacts/v2/properties";
```
Comment (ksvirkou) [inline src/Resources/ContactProperties.php]: Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)

File: src/Resources/ContactProperties.php [hunk 3]
```diff
@@ -107,7 +107,30 @@ public function delete($name)
      */
     public function getGroups($includeProperties = false)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/groups';
+        $endpoint = "https://api.hubapi.com/contacts/v2/groups";
```
Comment (ksvirkou) [inline src/Resources/ContactProperties.php]: Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)

File: src/Resources/ContactProperties.php [hunk 4]
```diff
@@ -131,7 +154,7 @@ public function getGroups($includeProperties = false)
      */
     public function createGroup(array $group)
     {
-        $endpoint = 'https://api.hubapi.com/contacts/v2/groups';
+        $endpoint = "https://api.hubapi.com/contacts/v2/groups";
```
Comment (ksvirkou) [inline src/Resources/ContactProperties.php]: Is it really necessary ?
You don’t use any params ...
it have to be wrapped ' (a single quotes)

File: src/Resources/ContactProperties.php [hunk 5]
```diff
@@ -107,7 +107,30 @@ public function delete($name)
...
+     *
+     * Returns all of the contact property group data for given name.
+     *
+     * @see https://developers.hubspot.com/docs/methods/contacts/v2/get_contact_property_group
+     *
+     * @param string $groupName
+     *
+     * @return \SevenShores\Hubspot\Http\Response
+     */
+    public function getGroupDetails($groupName, $includeProperties = false)
```
Comment (ksvirkou) [inline src/Resources/ContactProperties.php]: You can mark $groupName as string at this line 
v2 requires php 7 or higher

**Comment by ksvirkou-hubspot** [comment]
already added https://github.com/HubSpot/hubspot-php/pull/302

**Pull request closed without merging**